registry/configuring-registry-storage/configuring-registry-storage-vsphere.adoc

```diff
@@ -4,6 +4,20 @@ include::modules/common-attributes.adoc[]
 :context: configuring-registry-storage-vsphere
 toc::[]
 
+On platforms which do not provide shareable object storage, the OpenShift Image Registry Operator bootstraps itself as `Removed`. This allows `openshift-installer` to complete installations on these platform types.
```

Comment:
May need to walk this one back since the issue I have is specifically for bare metal.